### DISCUSSION TITLE
Some ideas

### DISCUSSION THREAD

**Discussion by g-braeunlich** [emilk/egui#272] [Ideas]
Here some extensions, I developed in my current project https://gitlab.com/g-braeunlich/dhe.
I bundled everything here: https://g-braeunlich.gitlab.io/egui_extras/widgets/index.html
The demo repo is at https://gitlab.com/g-braeunlich/egui_extras/

The demos are for inspiration (maybe start a cookbook?) / potential PR's (tell me if you want to integrate something) and a proof of concept of integrating egui itself in the docs (show the widgets next to their docs).

Also very happy for some feedback on how to improve things!

**Comment by emilk** [comment]
Oh my, the inline egui in the docs is amazing!
